Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Image/ImageTagSystemComponent.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+/**
+ * @file TextureTagSystemComponent.h
+ * @brief Contains the definition of the TextureTagSystemComponent that will allow to set all texture tags
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/std/containers/unordered_map.h>
+#include <Atom/RPI.Public/AssetTagBus.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
+        class ImageTagSystemComponent final
+            : public AZ::Component
+            , ImageTagBus::Handler
+        {
+        public:
+            AZ_COMPONENT(ImageTagSystemComponent, "{1AC446A3-6F20-4D5C-9C09-BB034C9188D5}");
+
+            static void Reflect(AZ::ReflectContext* context);
+            static void GetRequiredServices(AZ::ComponentDescriptor::DependencyArrayType& required);
+            static void GetProvidedServices(AZ::ComponentDescriptor::DependencyArrayType& provided);
+            static void GetDependentServices(AZ::ComponentDescriptor::DependencyArrayType& dependent);
+
+            ImageTagSystemComponent() = default;
+            ~ImageTagSystemComponent() override;
+
+            void Activate() override;
+            void Deactivate() override;
+
```

Comment:
nit: add comments for overrides to indicate the overrides for 
`// ImageTagBus::Handler overrides...`